consensus/consensus_msg_sender.go

```diff
@@ -65,7 +65,10 @@ func (sender *MessageSender) SendWithRetry(blockNum uint64, msgType msg_pb.Messa
 			sender.Retry(&msgRetry)
 		}()
 	}
-	return sender.host.SendMessageToGroups(groups, p2pMsg)
+	if immediate {
```

Comment:
it will send with retries if the consensus don't move forward, if moving forward normally, the retries won't happen. So for 100% committed, it's not always necessary to send out since the 67% commitited is already sent. 100% will be sent if consensus don't move forward (view change will happen, and next leader should better get the 100% sig)

Reply:
because 67% is already sent to the validators, so they can confirm the block already. 100% is only needed if there is a leader change and the next leader better have the 100% sig so more people's sig is included. If there is no leader change and the next consensus goes smoothly, the Committed msg retry will automatically be stopped.

Reply:
Added a new func DelayedSendWithRetry for this case.